### ISSUE TITLE
Make naming of functions external data source retrieval more consistent

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#529]
At the moment we've got `retrieve_ceda` and `retrieve_atmospheric`, maybe we should rename it `retrieve_icos_atmospheric` and just allow imports from `openghg.retrieve` instead of `openghg.retrieve.icos`? Not the highest priority thing but I think it'd make things tider.

**Comment by rt17603** [comment]
Could we take out of retrieve and put into its own module (or link more closely to standardise as "adding data")

**Comment by gareth-j** [comment]
Yeah, I can't think of a good submodule name at the moment but that could be a good idea.
